application/models/Entity/User.php

```diff
@@ -663,4 +663,16 @@ public function getCsvImports()
     {
         return $this->csv_imports;
     }
```

Comment:
In elevator we have sort of a two tier concept of models. There are the _model items (user_model), which are fat. Then there are the Doctrine shims, which tend to be skinny because they're mostly managed by the doctrine model generator based on the XML definitions. There are magic methods on the _models that call through to doctrine when needed. This is just a weird case because we're not actually bringing up a full user_model.